frontend/src/features/investigations/components/AddInvestigantion.test.tsx

```diff
@@ -0,0 +1,86 @@
+import { render as rtlRender, waitFor } from "@testing-library/react";
+import { beforeEach, describe, expect, test } from "vitest";
+
+import { ErrorBoundary } from "@/components/error/ErrorBoundary";
+import { ElectionLayout } from "@/components/layout/ElectionLayout";
+import { ElectionRequestHandler, ElectionStatusRequestHandler } from "@/testing/api-mocks/RequestHandlers";
+import { Providers } from "@/testing/Providers";
+import { server } from "@/testing/server";
+import { screen, setupTestRouter } from "@/testing/test-utils";
+
+import { AddInvestigantion } from "./AddInvestigantion";
+
+async function renderPage() {
+  const router = setupTestRouter([
+    {
+      path: "/elections/:electionId",
+      Component: ElectionLayout,
+      errorElement: <ErrorBoundary />,
+      children: [
+        {
+          path: "investigations",
+          children: [
+            {
+              path: "add",
+              Component: AddInvestigantion,
+              children: [
+                {
+                  path: ":pollingStationId",
+                  children: [
+                    {
+                      index: true,
+                      path: "reason",
+                      Component: () => "Reason stub",
+                    },
+                  ],
+                },
+              ],
+            },
+          ],
+        },
+      ],
+    },
+  ]);
+
+  await router.navigate("/elections/1/investigations/add");
+  rtlRender(<Providers router={router} />);
+
+  return router;
+}
+
+describe("AddInvestigantion", () => {
```

Comment:
```suggestion
describe("AddInvestigation", () => {
```
Also the filenames themselves and the Component have the same spelling error

Reply:
Fixed in c51a04a74e81fb260bccd3e569de131921f8fbb0